### ISSUE TITLE
Windows: not disable system proxy after restart os

### ISSUE THREAD

**Issue by myxomor** [hiddify/hiddify-app#1558]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

Hi! I have a problem. When I use hiddify connection and forget to turn off the connection to the server, then after rebooting the OS there is no internet. The system proxy settings remain enabled. Hiddify starts on OS startup, but does not make an automatic connection to the server. It turns out that all requests also go to the proxy address, but the proxy is not running.

### Minimal Reproducible Example (MRE)

1. With windows running, start hiddify.
2. Press the button to connect to the server.
3. Verify that the proxy is working.
4. Restart Windows.
5. After restarting, open the browser.
6. All sites show an error about inaccessibility.

### Expected Behavior

System shutdown will shut down hiddify and hiddify will return the system proxy settings as they were before startup.

### Version

2.1.1

### Platform/OS

Windows

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by goodboy34-tech** [comment]
Yes, it is. Infuriates incredibly.  And no internet until I go to Hiddify and hit the power button. It occurs on both win10 and win11.

**Comment by Andersonda1** [comment]
The same problem. After rebooting there is no Internet. The proxy in Windows settings remains on.

**Comment by stargeraaas** [comment]
Would you fix it in 2026? The same error...

**Comment by kfo1182** [comment]
As a makeshift treatment, I created a file, containing one line:

`Set-ItemProperty -Path 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Internet Settings' -Name ProxyEnable -Value 0`

This file is run by the task scheduler on logon, and it resolves the issue. It's hacky, but this doesn't seem to be Hiddify problem, it's more of a Windows problem. This cannot be reasonably fixed unless Hiddify runs on Windows startup. But even if it does run on startup, how should it know, if the proxy has been turned on by itself, or by another app, or manually?
